List/list.html

```diff
@@ -0,0 +1,140 @@
+                        <li class="list-item">Sub Title 1</li>
+                        <li class="list-item">Sub Title 2</li>
+                        <li class="list-item">Sub Title 3
+                            <ul class="list-bullet">
+                                <li class="list-item">Paragraph Text</li>
+                            </ul>
+                        </li>
+                    </ol>
+                </li>
+                <li class="list-item">Title 3
+                    <ol class="list-roman">
+                        <li class="list-item">Sub Title 1</li>
+                        <li class="list-item">Sub Title 2</li>
+                        <li class="list-item">Sub Title 3</li>
+                    </ol>
+                </li>
+            </ol>
+        </ul>
+    </div>
+    <div class="list-container">
+        <aside class="hero-drawer" id="Drawer">
+            <div class="drawer-header">
+                <h3 class="drawer-title">Title</h3>
+                <h6 class="drawer-subtext">subtext</h6>
+            </div>
+            <div class="drawer-content">
+                <nav class="drawer-content-list">
+                    <a href="#Drawer" class="list-item-drawer show">
+                        <span class="material-icons icons">inbox</span>
+                        <span class="item-title">Inbox</span>
+                    </a>
+                    <a href="#Drawer" class="list-item-drawer show">
+                        <span class="material-icons icons">star</span>
+                        <span class="item-title">Star</span>
+                    </a>
+                    <a href="#Drawer" class="list-item-drawer show">
+                        <span class="material-icons icons">send</span>
+                        <span class="item-title">Sent Mail</span>
+                    </a>
+                    <a href="#Drawer" class="list-item-drawer show">
+                        <span class="material-icons icons">drafts</span>
+                        <span class="item-title">Draft</span>
+                    </a>
+                </nav>
+            </div>
+        </aside>
+    </div>
+    <div class="list-container">
+        <h1>Stacked List</h1>
+        <div class="stacked-list">
+            <div class="d-flex align-center gap margin-16px">
+                <div class="img-container">
+                    <img class="avatar-image"
+                        src="https://static.toiimg.com/thumb/resizemode-4,msid-76729750,imgsize-249247,width-720/76729750.jpg"
+                        alt="image">
+                </div>
+                <div class="name d-flex align-center justify-center">
+                    <span>List Item 1</span>
+                    <input class="check-input" type="checkbox">
+                </div>
+            </div>
+            <div class="d-flex align-center gap margin-16px">
+                <div class="img-container">
+                    <img class="avatar-image"
```

Comment:
Have you used same avatar component from your CL?

Reply:
no, that has some other dimensions